core/cmd/starknet_keys_commands.go

```diff
@@ -13,7 +13,7 @@ type StarkNetKeyPresenter struct {
 
 // RenderTable implements TableRenderer
 func (p StarkNetKeyPresenter) RenderTable(rt RendererTable) error {
-	headers := []string{"ID", "Account Address", "Stark Public Key"}
+	headers := []string{"ID", "Stark Public Key"}
```

Comment:
nit/ Is it important to say `Stark` here?
```suggestion
	headers := []string{"ID", "Public Key"}
```

Reply:
nbd :shrug: